include/FileSystem.class.php

```diff
@@ -233,7 +233,7 @@ public static function fileOpen($filename, $mode, $use_include_path = false)
      *
      * @throws FileSystemException
      */
-    public static function fileClose(resource $handle)
+    public static function fileClose($handle)
```

Comment:
Why was resource removed? was that deprecated or something?

Reply:
`resource` is not valid typehint - it is not scalar nor class/interface.